PairedEndSingleSampleWf-fc-hg38.wdl

```diff
@@ -66,6 +68,9 @@ workflow PairedEndSingleSampleWorkflow {
   Int preemptible_tries
   Int agg_preemptible_tries
 
+  String docker
+  String gatk_docker
```

Comment:
What's the difference?  Should the top one be GOTC docker or something?

Reply:
It's also nice to put the gatk_path as a variable since it's different between GATK release and gitc.

Reply:
Renamed to gotc_docker and added path variables for both
done